src/background/listener/bee-api.listener.ts

```diff
@@ -4,9 +4,13 @@ import { SWARM_SESSION_ID_KEY, removeSwarmSessionIdFromUrl } from '../../utils/s
 
 export class BeeApiListener {
   private _beeApiUrl: string
+  private _globalPostageBatchEnabled: boolean
+  private _globalPostageBatchId: string
 
   public constructor(private storeObserver: StoreObserver) {
     this._beeApiUrl = 'http://localhost:1633'
+    this._globalPostageBatchEnabled = false
+    this._globalPostageBatchId = 'undefined' // it is not necessary to check later, if it is enabled it will insert
```

Comment:
What is not needed to check? The fact that it is undefined? The comment is unclear to me.

Reply:
it means regardless the correctness of the global postage batch id, it will be inserted into the corresponding postage batch id header if the global postage batch usage is enabled.
The Bee client can response in a better way about the invalid postage batch id, than it could happen by an extension response -> so it lets the error handling happen on the "server" side.